docs/index.rst

```diff
@@ -578,7 +578,7 @@ End clients (non-CP clients):
    .. sourcecode:: bash
 
       # First get the KAFKA_CLUSTER_ID
-      KAFKA_CLUSTER_ID=$(curl -s http://localhost:8091/v1/metadata/id | jq -r ".id")
+      KAFKA_CLUSTER_ID=$(curl -s -k https://localhost:8091/v1/metadata/id --cert scripts/security/mds.certificate.pem --key scripts/security/mds.key --tlsv1.2 --cacert scripts/security/snakeoil-ca-1.crt | jq -r ".id")
```

Comment:
Is this performing mTLS authentication to MDS? Is that necessary or recommended? Same question applies to other matching edits throughout this file.